client/getattr_freebsd.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018 The gVisor Authors.
```

Comment:
Similar?

Reply:
it's originally a gvisor thing so this one I want to leave as is. It's not much code.